lib/migration_helper.rb

```diff
@@ -1,54 +0,0 @@
-# Utility functions used in several migrations
```

Comment:
We can't just delete this file, as it is still used in lib/tasks/seed_skills.rake